js/src/apbct-public--5--external-forms.js

```diff
@@ -376,3 +377,99 @@ function sendAjaxCheckingFormData(form, prev, formOriginal) {
         }
     );
 }
+
+window.onload = function() {
+    setTimeout(function () {
+        catchDinamicRenderedForm()
+    }, 1500);
+}
+
+function catchDinamicRenderedForm() {
+    let forms = document.getElementsByTagName('form')
+
+    catchDinamicRenderedFormHandler(forms)
+
+    const frames = document.getElementsByTagName('iframe');
+    if ( frames.length > 0 ) {
+        for ( let j = 0; j < frames.length; j++ ) {
+            if ( frames[j].contentDocument == null ) {
+                continue;
+            }
+
+            const iframeForms = frames[j].contentDocument.forms;
+
+            if ( iframeForms.length === 0 ) {
+                return;
+            }
+
+            catchDinamicRenderedFormHandler(iframeForms, frames[j].contentDocument)
+        }
+    }
+}
+
+function catchDinamicRenderedFormHandler(forms, documentObject = document) {
+    let neededFormIds = [];
+    for (let form of forms) {
+        if (form.id.indexOf('hsForm') !== -1) {
+            neededFormIds.push(form.id)
+        }
+    }
+
+    for (let formId of neededFormIds) {
+        let form = documentObject.getElementById(formId);
+        form.onsubmit = sendAjaxCheckingDinamicFormData
+    }
+}
+
+/**
+ * Sending Ajax for checking form data on dinamic rendered form
+ */
+function sendAjaxCheckingDinamicFormData(form) {
+    form.preventDefault()
+    form = form.target
+
+    var force_action = document.createElement("input");
+    force_action.name = 'action';
+    force_action.value = 'cleantalk_force_ajax_check';
+    force_action.type = 'hidden';
+    form.appendChild(force_action);
+
+    // Get visible fields and set cookie
+    var visible_fields = {};
+    visible_fields[0] = apbct_collect_visible_fields(form);
+    apbct_visible_fields_set_cookie( visible_fields );
+
+    var data = {};
+    var elems = form.elements;
+    elems = Array.prototype.slice.call(elems);
+
+    elems.forEach( function( elem, y ) {
+        if( elem.name === '' ) {
+            data['input_' + y] = elem.value;
+        } else {
+            data[elem.name] = elem.value;
+        }
+    });
+
+    apbct_public_sendAJAX(
+        data,
+        {
+            async: false,
+            callback: function(result) {
+                if( result.apbct === undefined || ! +result.apbct.blocked ) {
+                    form.onsubmit = null
```

Comment:
Need to store the original submit handler and will return back it here

Reply:
Done.